### PR TITLE
Rework/split Parity configuration files

### PR DESCRIPTION
**Pull request by weilbith**
Closes: #546

### CODE REVIEW

File: chain/validator-config.toml [hunk 1]
```diff
@@ -6,36 +6,26 @@ keys_path = "/home/parity/.local/share/io.parity.ethereum/custom/keys"
...
 
 [rpc]
 interface = "all"
 
 [account]
 # File should contain passwords to unlock your accounts. One password per line.
-# password = ["/home/parity/.local/share/io.parity.ethereum/custom/pass.pwd"]
-# List of addresses to unlock for the duration of execution.
-# unlock = ["0xAddress"]
+password = ["/home/parity/.local/share/io.parity.ethereum/custom/pass.pwd"]
```
Comment (berndbohmeier) [inline chain/validator-config.toml]: Where is unlock?

File: chain/validator-config.toml [hunk 2]
```diff
@@ -6,36 +6,26 @@ keys_path = "/home/parity/.local/share/io.parity.ethereum/custom/keys"
...
 # Prepare a block to seal even when there are no miners connected.
 force_sealing = true
-# New transactions will not cause a new pending block to be created.
-reseal_on_txs = "none"
 # Minimum amount of Wei per GAS to be paid for a transaction to be accepted for mining. Overrides --usd-per-tx.
-min_gas_price = 1
+min_gas_price = 1000000000
 # Maximum number of transactions per sender in the queue. By default it's 1% of the entire queue, but not less than 16.
-tx_queue_per_sender = 100
+tx_queue_per_sender = 200
```
Comment (berndbohmeier) [inline chain/validator-config.toml]: We agreed yesterday, that 100 is fine.

File: chain/validator-config.toml (hunk 1 above)
Comment (weilbith) [reply chain/validator-config.toml]: You don't need this for a validator.
`git cat-file -p f7549b50f7aad00c2636925c25ef450e384e1e57:chain/parity_wrapper.sh` shows that this options was only added for the role of a participant.

File: chain/validator-config.toml (hunk 2 above)
Comment (weilbith) [reply chain/validator-config.toml]: Okay, then the information [in this comment](https://github.com/trustlines-protocol/blockchain/issues/546#issuecomment-549799507) was incorrect. I apologize for the typo. has been decreased again.

**Comment by berndbohmeier** [comment]
Superseeded by #552

**Pull request merged**